src/server/lib/jobs.ts

```diff
@@ -43,8 +61,31 @@ export const setupJobsUserInIDAPI = (
   // they try to sign in to the Jobs site for the first time.
   //
   // We can resolve both promises here because they are not dependent on each other.
-  return Promise.allSettled([
-    updateName(firstName, secondName, ip, sc_gu_u),
-    addToGroup(GroupCode.GRS, ip, sc_gu_u),
-  ]);
+  //
+  // We retry this operation up to the number of retries specified because the requests
+  // to IDAPI can intermittently fail.
+  try {
+    const updateResult = await Promise.allSettled([
+      updateName(firstName, secondName, ip, sc_gu_u),
+      addToGroup(GroupCode.GRS, ip, sc_gu_u),
+    ]);
+
+    return updateResult;
```

Comment:
could this just become?

```suggestion
    return await Promise.allSettled([
      updateName(firstName, secondName, ip, sc_gu_u),
      addToGroup(GroupCode.GRS, ip, sc_gu_u),
    ]);
```

Reply:
👍